README.md

```diff
@@ -1,90 +1,53 @@
-# custom-image-builder
+# k8s-nodeOS-builder
+> building custom image files for [docker based provisioning](https://github.com/FujitsuEnablingSoftwareTechnologyGmbH/k8s-docker-provisioner)
 
-## Prepere Openstack
+### Introduction
 
-Go to
-<https://github.com/FujitsuEnablingSoftwareTechnologyGmbH/devstack-vagrant>
+This project describes the process on how to create a base image that can be used for the [Docker Based Provisioning](https://github.com/FujitsuEnablingSoftwareTechnologyGmbH/k8s-docker-provisioner) project. Unlike other script based provisioning approaches, docker based provisioning will provision a kubernetes cluster by creating and launching only docker containers.
 
-Clone and run Openstack in Vagrand
+### Prerequisities
 
-### Upload reference image via glance
+The prerequisites for the docker based provisioning are:
+* [Docker](https://www.docker.com/)
+* Configuration of Docker called **Docker Bootstrap**.
+  * This particular configuration uses two docker daemons that are run in place of a typical, single daemon setup. The additional docker bootstrap daemon run etcd and flannel as docker containers, which in turn provide the network interface for the main docker daemon.
 
-```
-cd devstack-vagrant
-vagrant ssh
-
-sudo su - stack
-. devstack/openrc admin admin
-
-curl -L http://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-1603.qcow2.xz -O
-unxz CentOS-7-x86_64-GenericCloud-1603.qcow2.xz
-glance image-create --name centos7 --disk-format qcow2 --container-format bare --file CentOS-7-x86_64-GenericCloud-1603.qcow2 --is-public True
+### Supported formats
 
-exit
-```
+Currently supported OS:
+- [CentOS](https://www.centos.org/)
 
+Currently supported image type:
+*can easily be extended to other operating systems and image types*
+- [QCOW (OpenStack)](https://github.com/kenan435/k8s-nodeos-builder/blob/k8s-nodeos-builder/Openstack/README.md)
 
-### Openstack network configuration
-Packer requires well configured network to be able proceed all build steps.
-In file install.json you have to enter private network ID.
-Private network should be connected via router to public network.
 
+#### Kubernetes cluster with main and bootstrap docker daemons
+![Alt text](https://raw.githubusercontent.com/kenan435/k8s-nodeos-builder/k8s-nodeos-builder/k8s-docker.png "k8s Docker Based Provisioning")
 
 
-## Install Packer
-Packer is a tool developed by HashiCorp (the people behind Vagrant) to help you create identical cloud images for a variety of different environments. It also allows you to create image templates that are easy to version control and understand what happens during the image creation process.
+### FAQ
```

Comment:
I would suggest to move the FAQ to openstack/faq.md, since it is only openstack related